ingestion_server/ingestion_server/cleanup.py

```diff
@@ -48,9 +48,9 @@
 
 def _tag_blacklisted(tag):
     """ Tag is banned or contains a banned substring. """
-    if tag in TAG_BLACKLIST:
+    if tag in TAG_DENYLIST:
         return True
-    for blacklisted_substring in TAG_CONTAINS_BLACKLIST:
+    for blacklisted_substring in TAG_CONTAINS_DENYLIST:
         if blacklisted_substring in tag:
```

Comment:
To me, changing 'blacklist' to 'denylist' should come with changing 'blacklisted' to 'denylisted'.  This will make the resulting code a bit clearer.